crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -489,6 +495,44 @@ where
                                         continue;
                                     }
 
+                                    if !use_create_element
+                                        && i.sym == String::from("__source")
```

Comment:
`String::from("__source")` is very inefficient.
You can compare `JsWord` to `str` like `*i.sym === "__source ".`

Reply:
👍    1ed2c7b